happymonk-ai/Ckdir-frontend-
# Add validation for book demo form

What does this PR do?
• Added formik and yup validations for book demo form

How should this be manually tested?
• Clone this repo
• Check out this branch ft-homepage-update
• Run yarn install to install all packages
• Run yarn dev to start the development environment
• Navigate to "/register" and test the changes.
